CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# [7.17.0](https://github.com/dashevo/wallet-lib/compare/v7.16.1...v7.17.0) (2020-12-29)
+
+
+### Features
+
+* connect to testnet by default ([#210](https://github.com/dashevo/wallet-lib/issues/210))
+* update `dpp`, `dapi-client`, `dashcore-lib` ([#202](https://github.com/dashevo/wallet-lib/issues/202))
+* is locks subscription ([#206](https://github.com/dashevo/wallet-lib/issues/206))
+
+
+
```

Comment:
We need to add a breaking change that now wallet connects to test net by default